ocp4/profiles/moderate.profile

```diff
@@ -41,3 +41,5 @@ selections:
     - ocp_idp_no_htpasswd
     - ocp_allowed_registries_for_import
     - ocp_allowed_registries
+    # AU-9(2)
```

Comment:
I would stop tracking NIST refs in the profiles. The reason is for this is that there are usually more than 1 NIST reference to a rule and then the question is... which one is in the profile?